### ISSUE TITLE
Add favorite feature

### ISSUE THREAD

**Issue by BlockTester** [miguelfreitas/twister-html#70]
It occurred to me that while the feature is planned (or at least mocked up) there was no ticket to go with it. So here we go, the favorite feature needs to be implemented. I suppose it should do that in a privacy respecting but non-ephemeral way.

**Comment by talexand** [comment]
Agree, but I suggest making the favorite action not publicly visible. The psychological effect of visible 'likes' is a deplorable one, as many studies on Facebook have shown.
